lib/commons/odata/ODataClient.js

```diff
@@ -166,7 +186,26 @@ module.exports = class ODataClient {
     createdRecord.isNew = true;
     this.emitter.emit('data', messages.newMessageWithBody(createdRecord));
   }
+  async deleteObjectByField(msg) {
+    const objectType = this.cfg.objectType;
+    const fieldValue = msg.body.FieldsValue;
+    console.log(`Will perform delete to ${objectType} with field ${fieldValue}`);
+ 
+    const response = await this.restClient.makeRequest({
+    url: `${objectType}(${"'"+fieldValue+"'"})`,
+    method: 'GET',
+     body: null,
+    headers: {
+      'Content-Type': 'application/json'
+    }
+  });
+ 
+  console.log("response data "+ response);
```

Comment:
`JSON.stringify` is probably needed here to have this line log something meaningful.